addons/event/event_view.xml

```diff
@@ -95,18 +95,21 @@
                                 <group>
                                     <group>
                                         <field name="auto_confirm" invisible="1"/>
-                                        <field name="seats_min" attrs="{'invisible':[('auto_confirm','=', True)]}"/>
+                                        <field name="seats_min" attrs="{'readonly':[('auto_confirm','=', True)]}"/>
```

Comment:
Seems curious: you have a readonly field and you don't know why ... I had to see the code to understand that auto confirm make the field readonly. What is the reason of that ?